web/src/components/PageDetail.tsx

```diff
@@ -1,19 +1,27 @@
 import React, { ReactElement } from 'react'
 import styled from 'styled-components'
 
+import Link from './base/Link'
+
 const Identifier = styled.span`
   font-weight: 700;
 `
 
+const StyledLink = styled(Link)`
+  color: ${props => props.theme.colors.linkColor};
+  text-decoration: underline;
+`
```

Comment:
:upside_down_face: I think it would be cool if we could just use this style by passing one prop to the `Link` component as this is reused in a few places and it would make adjusting and using the style easier and guarantees consistent styling. I am thinking of a boolean prop called `styled` or `highlighted` or similar which uses this style if true and no styling if false.

Since this was not introduced in this PR, you don't have to do this obviously.